backend/models/postgis/user.py

```diff
@@ -151,7 +151,7 @@ def get_all_users(query: UserSearchQuery) -> UserSearchDTO:
             ]
             base = base.filter(User.mapping_level.in_(mapping_level_array))
         if query.username:
-            base = base.filter(User.username.ilike(query.username.lower() + "%"))
+            base = base.filter(User.username.contains(query.username.lower()))
```

Comment:
When I make a search for a character for example the letter 'a', this works well and returns usernames that contain it. 
<img width="583" alt="Screenshot 2020-12-03 at 11 49 49" src="https://user-images.githubusercontent.com/31903212/100986286-0b6fb400-355e-11eb-9251-cf75867300e0.png">
However if I try searching for a complete name, it does not return it. 
<img width="583" alt="Screenshot 2020-12-03 at 11 49 58" src="https://user-images.githubusercontent.com/31903212/100986365-26dabf00-355e-11eb-9031-eaf874415a20.png">
I wonder if we could just build on the previous implementation. Something like` base.filter(User.username.ilike(("%" + query.username + "%"))` so that we can get names that contain a character or the name itself.